fmriprep/workflows/bold.py

```diff
@@ -621,127 +638,151 @@ def init_bold_reference_wf(omp_nthreads, bold_file=None, name='bold_reference_wf
 
 
 # pylint: disable=R0914
-def init_bold_hmc_wf(metadata, bold_file_size_gb, ignore, omp_nthreads,
-                     name='bold_hmc_wf'):
+def init_bold_stc_wf(metadata, name='bold_stc_wf'):
     """
-    This workflow performs :abbr:`HMC (head motion correction)` over the input
+    This workflow performs :abbr:`STC (slice-timing correction)` over the input
     :abbr:`BOLD (blood-oxygen-level dependent)` image.
 
     .. workflow::
         :graph2use: orig
         :simple_form: yes
 
-        from fmriprep.workflows.bold import init_bold_hmc_wf
-        wf = init_bold_hmc_wf(
+        from fmriprep.workflows.bold import init_bold_stc_wf
+        wf = init_bold_stc_wf(
             metadata={"RepetitionTime": 2.0,
                       "SliceTiming": [0.0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9]},
-                      ignore=[],
-                      bold_file_size_gb=3,
-                      omp_nthreads=1)
+            )
 
     Parameters
 
         metadata : dict
             BIDS metadata for BOLD file
+        name : str
+            Name of workflow (default: ``bold_stc_wf``)
+
+    Inputs
+
+        bold_file
+            BOLD series NIfTI file
+        skip_vols
+            Number of non-steady-state volumes detected at beginning of ``bold_file``
+
+    Outputs
+
+        stc_file
+            Slice-timing corrected BOLD series NIfTI file
+
+    """
+    workflow = pe.Workflow(name=name)
+    inputnode = pe.Node(niu.IdentityInterface(fields=['bold_file', 'skip_vols']), name='inputnode')
+    outputnode = pe.Node(niu.IdentityInterface(fields=['stc_file']), name='outputnode')
+
+    LOGGER.info('Slice-timing correction will be included.')
+
+    def create_custom_slice_timing_file_func(metadata):
```

Comment:
Potentially a nice addition to nipype's `afni.TShift`, so you can just pass a list of values as an input, alternatively to setting `inputs.tpattern`. WDYT?

Reply:
:+1: But let's get this in first.

Reply:
Sure!